src/Controllers/GagePageController.js

```diff
@@ -690,6 +762,146 @@ var StreamStats;
+                                    this.value -= 365;
+                                }
+                                if (this.value == 285)
+                                    return 'Annual Average';
+                                return moment("2015 " + this.value, "YYYY DDD").format("MMM");
+                            }
+                        }
+                    },
+                    yAxis: {
+                        title: {
+                            text: 'Water Year'
+                        },
+                        custom: {
+                            allowNegativeLog: true
+                        }
+                    },
+                    colorAxis: logOrLinear(this.dailyValuesOnly),
+                    series: [{
+                            name: 'Daily Streamflow',
+                            pixelSpacing: null,
+                            borderWidth: 0,
+                            borderColor: 'white',
+                            type: 'heatmap',
+                            data: this.formattedDailyPlusAvg[0],
+                            tooltip: {
+                                headerFormat: '<b>Daily Streamflow</b>',
+                                pointFormatter: function () {
+                                    if (this.formattedDailyPlusAvg !== null) {
+                                        var year = this.y;
+                                        var doy = this.x;
+                                        if (doy > 366) {
+                                            doy -= 366;
+                                        }
+                                        ;
+                                        if (doy > 274) {
+                                            year -= 1;
+                                        }
+                                        ;
+                                        if (isLeapYear(year) == false && doy > 59) {
+                                            doy -= 1;
+                                        }
+                                        ;
+                                        var fullDate = new Date(year, 0, doy);
+                                        var UTCday = fullDate.getUTCDate();
+                                        var month = fullDate.getUTCMonth();
+                                        month += 1;
+                                        var formattedUTCDate = month + '/' + UTCday + '/' + year;
+                                        var waterYear = year;
+                                        if (month > 9) {
+                                            waterYear += 1;
+                                        }
+                                        ;
+                                        if (doy > 282 && doy < 293)
+                                            return '</b><br>Water Year Average Value: <b>' + this.value.toFixed(2) + ' ft³/s</b><br>Water Year: <b>' + waterYear;
```

Comment:
```suggestion
                                            return '</b><br>Water Year: <b>' + waterYear<br>Water Year Average Value: <b>' + this.value.toFixed(2) + ' ft³/s</b>;
```
Suggestion to put Water Year above the Water Year Average Value

Reply:
I am still seeing Water Year Average Value above Water Year
![image](https://user-images.githubusercontent.com/40237491/222547548-9d618e93-7220-47ba-9ef8-8bac69723193.png)

Reply:
Whoops I thought that one would automatically fix with the GitHub commit suggestion button. Just fixed it locally!